src/js/components/Select/Select.js

```diff
@@ -138,7 +142,11 @@ class Select extends Component {
               flex={false}
               style={{ minWidth: 'auto' }}
             >
-              <SelectIcon color='brand' size={size} />
+              <ThemeContext.Extend
+                value={{ color: theme.select.iconColor || colorForName('brand', theme) }}
```

Comment:
I'm wondering how we should handle light vs. dark context here. Perhaps we should have `theme.select.icons.color.dark` and `theme.select.icons.color.light`? This might help with https://github.com/grommet/grommet/issues/1982 too.

Reply:
yeah agree we should have two colors for Select icon color. let me change it.